src/js/AppStore.js

```diff
@@ -55,11 +55,25 @@ class AppStore extends React.Component {
                     return;
                 }
 
+                var style = {};
+                if (manifest.category === 'WEB_VIEW') {
+                    style = {
+                        position: 'absolute',
+                        width: '960px',
```

Comment:
The size of the web view needs to take into account the dynamic height and width parameters set in the config and used by the scss files. It might be cleaner to define these styles in an scss file and pass a className within the js code instead.

Reply:
Good point, I applied these changes here: https://github.com/smartdevicelink/generic_hmi/pull/291/commits/ab9eaef5a91b50c9af0b59a089cf96ae4e7909f5